@types/react-test-renderer 15 introduced a breaking bug

**Version:** 0.2.6

**Code**

Error when running `tsc` commands:

```
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3420,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'a' must be of type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>', but here has type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3421,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'abbr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3422,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'address' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3423,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'area' must be of type 'DetailedHTMLProps<AreaHTMLAttributes<HTMLAreaElement>, HTMLAreaElement>', but here has type 'DetailedHTMLProps<AreaHTMLAttributes<HTMLAreaElement>, HTMLAreaElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3424,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'article' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3425,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'aside' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3426,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'audio' must be of type 'DetailedHTMLProps<AudioHTMLAttributes<HTMLAudioElement>, HTMLAudioElement>', but here has type 'DetailedHTMLProps<AudioHTMLAttributes<HTMLAudioElement>, HTMLAudioElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3427,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'b' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3428,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'base' must be of type 'DetailedHTMLProps<BaseHTMLAttributes<HTMLBaseElement>, HTMLBaseElement>', but here has type 'DetailedHTMLProps<BaseHTMLAttributes<HTMLBaseElement>, HTMLBaseElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3429,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'bdi' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3430,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'bdo' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3431,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'big' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3432,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'blockquote' must be of type 'DetailedHTMLProps<BlockquoteHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<BlockquoteHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3433,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'body' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLBodyElement>, HTMLBodyElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLBodyElement>, HTMLBodyElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3434,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'br' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLBRElement>, HTMLBRElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLBRElement>, HTMLBRElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3435,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'button' must be of type 'DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement>', but here has type 'DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3436,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'canvas' must be of type 'DetailedHTMLProps<CanvasHTMLAttributes<HTMLCanvasElement>, HTMLCanvasElement>', but here has type 'DetailedHTMLProps<CanvasHTMLAttributes<HTMLCanvasElement>, HTMLCanvasElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3437,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'caption' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3438,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'cite' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3439,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'code' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3440,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'col' must be of type 'DetailedHTMLProps<ColHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>', but here has type 'DetailedHTMLProps<ColHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3441,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'colgroup' must be of type 'DetailedHTMLProps<ColgroupHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>', but here has type 'DetailedHTMLProps<ColgroupHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3442,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'data' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3443,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'datalist' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLDataListElement>, HTMLDataListElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLDataListElement>, HTMLDataListElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3444,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'dd' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3445,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'del' must be of type 'DetailedHTMLProps<DelHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<DelHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3446,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'details' must be of type 'DetailedHTMLProps<DetailsHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<DetailsHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3447,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'dfn' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3448,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'dialog' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3449,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'div' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3450,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'dl' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLDListElement>, HTMLDListElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLDListElement>, HTMLDListElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3451,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'dt' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3452,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'em' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3453,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'embed' must be of type 'DetailedHTMLProps<EmbedHTMLAttributes<HTMLEmbedElement>, HTMLEmbedElement>', but here has type 'DetailedHTMLProps<EmbedHTMLAttributes<HTMLEmbedElement>, HTMLEmbedElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3454,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'fieldset' must be of type 'DetailedHTMLProps<FieldsetHTMLAttributes<HTMLFieldSetElement>, HTMLFieldSetElement>', but here has type 'DetailedHTMLProps<FieldsetHTMLAttributes<HTMLFieldSetElement>, HTMLFieldSetElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3455,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'figcaption' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3456,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'figure' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3457,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'footer' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3458,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'form' must be of type 'DetailedHTMLProps<FormHTMLAttributes<HTMLFormElement>, HTMLFormElement>', but here has type 'DetailedHTMLProps<FormHTMLAttributes<HTMLFormElement>, HTMLFormElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3459,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'h1' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3460,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'h2' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3461,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'h3' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3462,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'h4' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3463,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'h5' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3464,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'h6' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3465,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'head' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadElement>, HTMLHeadElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadElement>, HTMLHeadElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3466,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'header' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3467,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'hgroup' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3468,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'hr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHRElement>, HTMLHRElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHRElement>, HTMLHRElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3469,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'html' must be of type 'DetailedHTMLProps<HtmlHTMLAttributes<HTMLHtmlElement>, HTMLHtmlElement>', but here has type 'DetailedHTMLProps<HtmlHTMLAttributes<HTMLHtmlElement>, HTMLHtmlElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3470,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'i' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3471,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'iframe' must be of type 'DetailedHTMLProps<IframeHTMLAttributes<HTMLIFrameElement>, HTMLIFrameElement>', but here has type 'DetailedHTMLProps<IframeHTMLAttributes<HTMLIFrameElement>, HTMLIFrameElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3472,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'img' must be of type 'DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>', but here has type 'DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3473,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'input' must be of type 'DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>', but here has type 'DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3474,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'ins' must be of type 'DetailedHTMLProps<InsHTMLAttributes<HTMLModElement>, HTMLModElement>', but here has type 'DetailedHTMLProps<InsHTMLAttributes<HTMLModElement>, HTMLModElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3475,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'kbd' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3476,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'keygen' must be of type 'DetailedHTMLProps<KeygenHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<KeygenHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3477,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'label' must be of type 'DetailedHTMLProps<LabelHTMLAttributes<HTMLLabelElement>, HTMLLabelElement>', but here has type 'DetailedHTMLProps<LabelHTMLAttributes<HTMLLabelElement>, HTMLLabelElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3478,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'legend' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLLegendElement>, HTMLLegendElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLLegendElement>, HTMLLegendElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3479,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'li' must be of type 'DetailedHTMLProps<LiHTMLAttributes<HTMLLIElement>, HTMLLIElement>', but here has type 'DetailedHTMLProps<LiHTMLAttributes<HTMLLIElement>, HTMLLIElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3480,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'link' must be of type 'DetailedHTMLProps<LinkHTMLAttributes<HTMLLinkElement>, HTMLLinkElement>', but here has type 'DetailedHTMLProps<LinkHTMLAttributes<HTMLLinkElement>, HTMLLinkElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3481,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'main' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3482,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'map' must be of type 'DetailedHTMLProps<MapHTMLAttributes<HTMLMapElement>, HTMLMapElement>', but here has type 'DetailedHTMLProps<MapHTMLAttributes<HTMLMapElement>, HTMLMapElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3483,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'mark' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3484,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'menu' must be of type 'DetailedHTMLProps<MenuHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<MenuHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3485,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'menuitem' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3486,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'meta' must be of type 'DetailedHTMLProps<MetaHTMLAttributes<HTMLMetaElement>, HTMLMetaElement>', but here has type 'DetailedHTMLProps<MetaHTMLAttributes<HTMLMetaElement>, HTMLMetaElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3487,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'meter' must be of type 'DetailedHTMLProps<MeterHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<MeterHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3488,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'nav' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3489,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'noindex' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3490,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'noscript' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3491,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'object' must be of type 'DetailedHTMLProps<ObjectHTMLAttributes<HTMLObjectElement>, HTMLObjectElement>', but here has type 'DetailedHTMLProps<ObjectHTMLAttributes<HTMLObjectElement>, HTMLObjectElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3492,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'ol' must be of type 'DetailedHTMLProps<OlHTMLAttributes<HTMLOListElement>, HTMLOListElement>', but here has type 'DetailedHTMLProps<OlHTMLAttributes<HTMLOListElement>, HTMLOListElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3493,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'optgroup' must be of type 'DetailedHTMLProps<OptgroupHTMLAttributes<HTMLOptGroupElement>, HTMLOptGroupElement>', but here has type 'DetailedHTMLProps<OptgroupHTMLAttributes<HTMLOptGroupElement>, HTMLOptGroupElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3494,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'option' must be of type 'DetailedHTMLProps<OptionHTMLAttributes<HTMLOptionElement>, HTMLOptionElement>', but here has type 'DetailedHTMLProps<OptionHTMLAttributes<HTMLOptionElement>, HTMLOptionElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3495,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'output' must be of type 'DetailedHTMLProps<OutputHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<OutputHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3496,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'p' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLParagraphElement>, HTMLParagraphElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLParagraphElement>, HTMLParagraphElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3497,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'param' must be of type 'DetailedHTMLProps<ParamHTMLAttributes<HTMLParamElement>, HTMLParamElement>', but here has type 'DetailedHTMLProps<ParamHTMLAttributes<HTMLParamElement>, HTMLParamElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3498,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'picture' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3499,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'pre' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLPreElement>, HTMLPreElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLPreElement>, HTMLPreElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3500,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'progress' must be of type 'DetailedHTMLProps<ProgressHTMLAttributes<HTMLProgressElement>, HTMLProgressElement>', but here has type 'DetailedHTMLProps<ProgressHTMLAttributes<HTMLProgressElement>, HTMLProgressElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3501,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'q' must be of type 'DetailedHTMLProps<QuoteHTMLAttributes<HTMLQuoteElement>, HTMLQuoteElement>', but here has type 'DetailedHTMLProps<QuoteHTMLAttributes<HTMLQuoteElement>, HTMLQuoteElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3502,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'rp' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3503,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'rt' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3504,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'ruby' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3505,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 's' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3506,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'samp' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3507,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'script' must be of type 'DetailedHTMLProps<ScriptHTMLAttributes<HTMLScriptElement>, HTMLScriptElement>', but here has type 'DetailedHTMLProps<ScriptHTMLAttributes<HTMLScriptElement>, HTMLScriptElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3508,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'section' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3509,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'select' must be of type 'DetailedHTMLProps<SelectHTMLAttributes<HTMLSelectElement>, HTMLSelectElement>', but here has type 'DetailedHTMLProps<SelectHTMLAttributes<HTMLSelectElement>, HTMLSelectElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3510,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'small' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3511,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'source' must be of type 'DetailedHTMLProps<SourceHTMLAttributes<HTMLSourceElement>, HTMLSourceElement>', but here has type 'DetailedHTMLProps<SourceHTMLAttributes<HTMLSourceElement>, HTMLSourceElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3512,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'span' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLSpanElement>, HTMLSpanElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLSpanElement>, HTMLSpanElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3513,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'strong' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3514,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'style' must be of type 'DetailedHTMLProps<StyleHTMLAttributes<HTMLStyleElement>, HTMLStyleElement>', but here has type 'DetailedHTMLProps<StyleHTMLAttributes<HTMLStyleElement>, HTMLStyleElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3515,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'sub' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3516,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'summary' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3517,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'sup' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3518,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'table' must be of type 'DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>', but here has type 'DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3519,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'tbody' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3520,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'td' must be of type 'DetailedHTMLProps<TdHTMLAttributes<HTMLTableDataCellElement>, HTMLTableDataCellElement>', but here has type 'DetailedHTMLProps<TdHTMLAttributes<HTMLTableDataCellElement>, HTMLTableDataCellElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3521,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'textarea' must be of type 'DetailedHTMLProps<TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement>', but here has type 'DetailedHTMLProps<TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3522,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'tfoot' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3523,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'th' must be of type 'DetailedHTMLProps<ThHTMLAttributes<HTMLTableHeaderCellElement>, HTMLTableHeaderCellElement>', but here has type 'DetailedHTMLProps<ThHTMLAttributes<HTMLTableHeaderCellElement>, HTMLTableHeaderCellElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3524,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'thead' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3525,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'time' must be of type 'DetailedHTMLProps<TimeHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<TimeHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3526,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'title' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTitleElement>, HTMLTitleElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTitleElement>, HTMLTitleElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3527,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'tr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableRowElement>, HTMLTableRowElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableRowElement>, HTMLTableRowElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3528,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'track' must be of type 'DetailedHTMLProps<TrackHTMLAttributes<HTMLTrackElement>, HTMLTrackElement>', but here has type 'DetailedHTMLProps<TrackHTMLAttributes<HTMLTrackElement>, HTMLTrackElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3529,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'u' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3530,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'ul' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLUListElement>, HTMLUListElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLUListElement>, HTMLUListElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3531,13): error TS2403: Subsequent variable declarations must have the same type.  Variable '"var"' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3532,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'video' must be of type 'DetailedHTMLProps<VideoHTMLAttributes<HTMLVideoElement>, HTMLVideoElement>', but here has type 'DetailedHTMLProps<VideoHTMLAttributes<HTMLVideoElement>, HTMLVideoElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3533,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'wbr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3536,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'svg' must be of type 'SVGProps<SVGSVGElement>', but here has type 'SVGProps<SVGSVGElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3538,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'animate' must be of type 'SVGProps<SVGElement>', but here has type 'SVGProps<SVGElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3539,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'animateTransform' must be of type 'SVGProps<SVGElement>', but here has type 'SVGProps<SVGElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3540,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'circle' must be of type 'SVGProps<SVGCircleElement>', but here has type 'SVGProps<SVGCircleElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3541,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'clipPath' must be of type 'SVGProps<SVGClipPathElement>', but here has type 'SVGProps<SVGClipPathElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3542,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'defs' must be of type 'SVGProps<SVGDefsElement>', but here has type 'SVGProps<SVGDefsElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3543,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'desc' must be of type 'SVGProps<SVGDescElement>', but here has type 'SVGProps<SVGDescElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3544,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'ellipse' must be of type 'SVGProps<SVGEllipseElement>', but here has type 'SVGProps<SVGEllipseElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3545,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feBlend' must be of type 'SVGProps<SVGFEBlendElement>', but here has type 'SVGProps<SVGFEBlendElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3546,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feColorMatrix' must be of type 'SVGProps<SVGFEColorMatrixElement>', but here has type 'SVGProps<SVGFEColorMatrixElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3547,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feComponentTransfer' must be of type 'SVGProps<SVGFEComponentTransferElement>', but here has type 'SVGProps<SVGFEComponentTransferElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3548,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feComposite' must be of type 'SVGProps<SVGFECompositeElement>', but here has type 'SVGProps<SVGFECompositeElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3549,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feConvolveMatrix' must be of type 'SVGProps<SVGFEConvolveMatrixElement>', but here has type 'SVGProps<SVGFEConvolveMatrixElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3550,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feDiffuseLighting' must be of type 'SVGProps<SVGFEDiffuseLightingElement>', but here has type 'SVGProps<SVGFEDiffuseLightingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3551,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feDisplacementMap' must be of type 'SVGProps<SVGFEDisplacementMapElement>', but here has type 'SVGProps<SVGFEDisplacementMapElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3552,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feDistantLight' must be of type 'SVGProps<SVGFEDistantLightElement>', but here has type 'SVGProps<SVGFEDistantLightElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3553,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feFlood' must be of type 'SVGProps<SVGFEFloodElement>', but here has type 'SVGProps<SVGFEFloodElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3554,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feFuncA' must be of type 'SVGProps<SVGFEFuncAElement>', but here has type 'SVGProps<SVGFEFuncAElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3555,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feFuncB' must be of type 'SVGProps<SVGFEFuncBElement>', but here has type 'SVGProps<SVGFEFuncBElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3556,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feFuncG' must be of type 'SVGProps<SVGFEFuncGElement>', but here has type 'SVGProps<SVGFEFuncGElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3557,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feFuncR' must be of type 'SVGProps<SVGFEFuncRElement>', but here has type 'SVGProps<SVGFEFuncRElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3558,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feGaussianBlur' must be of type 'SVGProps<SVGFEGaussianBlurElement>', but here has type 'SVGProps<SVGFEGaussianBlurElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3559,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feImage' must be of type 'SVGProps<SVGFEImageElement>', but here has type 'SVGProps<SVGFEImageElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3560,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feMerge' must be of type 'SVGProps<SVGFEMergeElement>', but here has type 'SVGProps<SVGFEMergeElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3561,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feMergeNode' must be of type 'SVGProps<SVGFEMergeNodeElement>', but here has type 'SVGProps<SVGFEMergeNodeElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3562,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feMorphology' must be of type 'SVGProps<SVGFEMorphologyElement>', but here has type 'SVGProps<SVGFEMorphologyElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3563,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feOffset' must be of type 'SVGProps<SVGFEOffsetElement>', but here has type 'SVGProps<SVGFEOffsetElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3564,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'fePointLight' must be of type 'SVGProps<SVGFEPointLightElement>', but here has type 'SVGProps<SVGFEPointLightElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3565,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feSpecularLighting' must be of type 'SVGProps<SVGFESpecularLightingElement>', but here has type 'SVGProps<SVGFESpecularLightingElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3566,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feSpotLight' must be of type 'SVGProps<SVGFESpotLightElement>', but here has type 'SVGProps<SVGFESpotLightElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3567,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feTile' must be of type 'SVGProps<SVGFETileElement>', but here has type 'SVGProps<SVGFETileElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3568,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'feTurbulence' must be of type 'SVGProps<SVGFETurbulenceElement>', but here has type 'SVGProps<SVGFETurbulenceElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3569,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'filter' must be of type 'SVGProps<SVGFilterElement>', but here has type 'SVGProps<SVGFilterElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3570,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'foreignObject' must be of type 'SVGProps<SVGForeignObjectElement>', but here has type 'SVGProps<SVGForeignObjectElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3571,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'g' must be of type 'SVGProps<SVGGElement>', but here has type 'SVGProps<SVGGElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3572,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'image' must be of type 'SVGProps<SVGImageElement>', but here has type 'SVGProps<SVGImageElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3573,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'line' must be of type 'SVGProps<SVGLineElement>', but here has type 'SVGProps<SVGLineElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3574,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'linearGradient' must be of type 'SVGProps<SVGLinearGradientElement>', but here has type 'SVGProps<SVGLinearGradientElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3575,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'marker' must be of type 'SVGProps<SVGMarkerElement>', but here has type 'SVGProps<SVGMarkerElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3576,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'mask' must be of type 'SVGProps<SVGMaskElement>', but here has type 'SVGProps<SVGMaskElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3577,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'metadata' must be of type 'SVGProps<SVGMetadataElement>', but here has type 'SVGProps<SVGMetadataElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3578,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'path' must be of type 'SVGProps<SVGPathElement>', but here has type 'SVGProps<SVGPathElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3579,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'pattern' must be of type 'SVGProps<SVGPatternElement>', but here has type 'SVGProps<SVGPatternElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3580,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'polygon' must be of type 'SVGProps<SVGPolygonElement>', but here has type 'SVGProps<SVGPolygonElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3581,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'polyline' must be of type 'SVGProps<SVGPolylineElement>', but here has type 'SVGProps<SVGPolylineElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3582,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'radialGradient' must be of type 'SVGProps<SVGRadialGradientElement>', but here has type 'SVGProps<SVGRadialGradientElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3583,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'rect' must be of type 'SVGProps<SVGRectElement>', but here has type 'SVGProps<SVGRectElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3584,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'stop' must be of type 'SVGProps<SVGStopElement>', but here has type 'SVGProps<SVGStopElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3585,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'switch' must be of type 'SVGProps<SVGSwitchElement>', but here has type 'SVGProps<SVGSwitchElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3586,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'symbol' must be of type 'SVGProps<SVGSymbolElement>', but here has type 'SVGProps<SVGSymbolElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3587,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'text' must be of type 'SVGProps<SVGTextElement>', but here has type 'SVGProps<SVGTextElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3588,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'textPath' must be of type 'SVGProps<SVGTextPathElement>', but here has type 'SVGProps<SVGTextPathElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3589,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'tspan' must be of type 'SVGProps<SVGTSpanElement>', but here has type 'SVGProps<SVGTSpanElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3590,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'use' must be of type 'SVGProps<SVGUseElement>', but here has type 'SVGProps<SVGUseElement>'.
node_modules/@types/react-test-renderer/node_modules/@types/react/index.d.ts(3591,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'view' must be of type 'SVGProps<SVGViewElement>', but here has type 'SVGProps<SVGViewElement>'.
node_modules/@types/react/index.d.ts(3454,19): error TS2320: Interface 'Element' cannot simultaneously extend types 'ReactElement<any>' and 'ReactElement<any>'.
  Named property 'type' of types 'ReactElement<any>' and 'ReactElement<any>' are not identical.
node_modules/@types/react/index.d.ts(3455,19): error TS2320: Interface 'ElementClass' cannot simultaneously extend types 'Component<any, {}>' and 'Component<any, {}>'.
  Named property 'refs' of types 'Component<any, {}>' and 'Component<any, {}>' are not identical.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @jedmao/react-bem@0.2.6 build: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @jedmao/react-bem@0.2.6 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/atom-danielflores/.npm/_logs/2017-10-09T23_16_53_879Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @jedmao/react-bem@0.2.6 pretest: `npm run build`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @jedmao/react-bem@0.2.6 pretest script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/atom-danielflores/.npm/_logs/2017-10-09T23_16_53_904Z-debug.log
```

**Expected behavior:**

No error.

**Actual behavior:**

Error

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,6 @@
+## 0.2.7
+- **Fix:** Upgrade @types/react-test-renderer 16 to resolve a bug introduced in 15. (fixes [`#21`](https://github.com/jedmao/react-bem/issues/21)).
+
 ## 0.2.6
 - **Fix:** Ensure `resolveBEMNode` can render nested block components (fixes [`#7`](https://github.com/jedmao/react-bem/issues/7)).
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@jedmao/react-bem",
-  "version": "0.2.6",
+  "version": "0.2.7",
   "description": "BEM helper functions and HOCs for React.",
   "keywords": [
     "react",
@@ -50,7 +50,7 @@
     "@types/prop-types": "^15.5.2",
     "@types/react": "^16.0.9",
     "@types/react-dom": "^16.0.0",
-    "@types/react-test-renderer": "^15.5.4",
+    "@types/react-test-renderer": "^16.0.0",
     "bem-helpers": "^0.9.0",
     "bem-join": "^0.2.0"
   },

